example/test_calendar.yaml

```diff
@@ -1,20 +1,34 @@
 name: Test Calendar
 timezone: America/Los_Angeles
+# Include another list of events in with these.  Note you can also do
+# this from the command line.
+#include:
+#  - other_calender.ics
```

Comment:
```suggestion
# # Include another list of events in with these.  Note you can also do
# # this from the command line.
# include:
#   - other_calender.ics
```

This way, when you "remove comments" with most editors, the comment will remain intact.